src/apis/objkt/utils.ts

```diff
@@ -26,7 +26,18 @@ export const transformCollectiblesArray = (
       : [];
     const lastPrice = buyEvents.find(event => event.price_xtz !== null);
     const lastPriceCurrencyId = lastPrice?.currency_id ?? 1;
-    const correctOffers = token.offers_active.filter(offer => offer.buyer_address !== selectedPublicKey);
+    const correctOffers = token.offers_active
+      .map(item => ({
+        ...item,
```

Comment:
I think it's bad practice here, because we left all snake_case properties in this object..